boards/stm32-sr3.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __BOARD_H__
+#define __BOARD_H__
+
+#include <stdbool.h>
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/gpio.h>
+
+/*
+ * Board definitions for "redpill" and "bluepill" boards
```

Comment:
Please update the comments

Reply:
This board is not common, I will remove this and send another pull request.